src/main/java/woohakdong/server/api/service/item/ItemService.java

```diff
@@ -0,0 +1,76 @@
+package woohakdong.server.api.service.item;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import woohakdong.server.api.controller.item.dto.ItemListResponse;
+import woohakdong.server.api.controller.item.dto.ItemRegisterRequest;
+import woohakdong.server.api.controller.item.dto.ItemRegisterResponse;
+import woohakdong.server.common.exception.CustomErrorInfo;
+import woohakdong.server.common.exception.CustomException;
+import woohakdong.server.domain.club.Club;
+import woohakdong.server.domain.club.ClubRepository;
+import woohakdong.server.domain.item.Item;
+import woohakdong.server.domain.item.ItemRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static woohakdong.server.common.exception.CustomErrorInfo.CLUB_NOT_FOUND;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class ItemService {
+
+    private final ItemRepository itemRepository;
+    private final ClubRepository clubRepository;
+
+    public ItemRegisterResponse registerItem(Long clubId, ItemRegisterRequest request) {
+        Club club = clubRepository.findById(clubId)
+                .orElseThrow(() -> new CustomException(CLUB_NOT_FOUND));
+
```

Comment:
이 메소드가 DB에 새로운 데이터 저장하는 것처럼 보이는데, 맞을까요?
맞다면 아래처럼 작성하여 (readOnly = true) 옵션을 false로 해주셔야할 것 같습니다!
```suggestion
    @Transactional
    public ItemRegisterResponse registerItem(Long clubId, ItemRegisterRequest request) {
        Club club = clubRepository.findById(clubId)
                .orElseThrow(() -> new CustomException(CLUB_NOT_FOUND));

```

Reply:
앗 제가 착각했네요 실수 지적 감사드립니다.